README.rst

```diff
@@ -124,14 +124,39 @@ To configure autoscaling:
 1. Set the METACI_MAX_WORKERS setting to the maximum number of dynos you'd like to scale up to.
 2. Set the METACI_WORKER_RESERVE setting to the number of dynos you'd like to reserve for high-priority builds. (Optional; defaults to 1.)
 3. Set up a Heroku user with access to this app, and create an authorization token using ``heroku authorizations:create``. Set the HEROKU_TOKEN setting to this authorization token.
-4. Set the AUTOSCALERS setting as a dict in the following format: {'app_name : {'app_name': name, 'worker_type': type, 'max_workers': METACI_MAX_WORKERS, 'worker_reserve': METACI_WORKER_RESERVE, 'queues': [list of queues]}}. You may list more than one Heroku app in the in AUTOSCALERS setting.
+4. Set the AUTOSCALERS setting as a dict in the following format: {'app_name : {'app_name': name, 'worker_type': type, 'max_workers': METACI_MAX_WORKERS, 'worker_reserve': METACI_WORKER_RESERVE, 'queues': [list of queues]}}. You may list more than one Heroku app in the in AUTOSCALERS setting and MetaCI will scale them all up and down at the same time.
     1. app_name - The name of the Heroku App.
     2. queues - a list of redis queues to monitor
     3. worker_type - The name of the worker dynos allocated for the given queues.
     4. max_workers - See METACI_MAX_WORKERS
     5. worker_reserve - See METACI_WORKER_RESERVE
 
 
+One-Off Builds
+~~~~~~~~~~~~~~
+
+In some environments, such as Heroku, it is helpful to run builds in
+environments which are spun up for just a single build. In Heroku, builds
+created in this way will not share their finite lifespan (24 hours) with
+previous builds. You might also want single-environment builds for
+security reasons, to ensure that no artifacts from one build can
+be snooped in another.
```

Comment:
One-off dynos do not help in that regard as long as they still have database credentials.

Reply:
The other benefit that's worth mentioning is that one-off dynos won't get restarted when the app is released.